### PR TITLE
Add daily java and docker release jobs

### PR DESCRIPTION
**Pull request by JPWKU**
These jobs run daily and push artifacts to the
Nexus staging repository.

Signed-off-by: Jeremy Phelps <jphelps@linuxfoundation.org>

### CODE REVIEW

**Review by trcox — CHANGES_REQUESTED** [review]
Need daily job definitions for the support/* jobs

**Comment by JPWKU** [comment]
@trcox Updated to include support/* jobs

**Review by trcox — APPROVED** [review]
LGTM

File: jjb/edgex-templates-docker.yaml [hunk 1]
```diff
@@ -213,5 +213,5 @@
       # Provided all steps have already passed, push the docker image
       - shell: !include-raw: ../shell/docker-push.sh
 
-      - triggers:
-          - timed: 'H 11 * * *'
+    triggers:
```
Comment (JPWKU) [inline jjb/edgex-templates-docker.yaml:216]: @tykeal I missed this

Comment (JPWKU) [reply jjb/edgex-templates-docker.yaml:216]: It was scoped inside the builder section

**Review by tykeal — APPROVED** [review]

**Pull request merged**